### DISCUSSION TITLE
Defining the folder where the batch.log file is created

### DISCUSSION THREAD

**Discussion by shizzlemaster** [ansys/pyaedt#5060] [Q&A] [answered]
Hello community,

I am using pyaedt to write a simulation script that is called by another application and I have the following issue:

The batch.log is created in the folder of my external application e.g. .../my_application/

This is not wanted in my case and I want to specify the directory where the batch.log is created but I do not know how.

Can anyone help me with this issue?

Thanks in advance

Marcus

**Comment by Samuelopez-ansys** [comment, marked as answer]
@shizzlemaster For now the file is created in the directory where you have the script running. I will create an Issue, because this feature is not implemented.

**Reply by shizzlemaster** [reply]
Thank you this answers my question. Got the file where I wanted it with your help. Not the most elegant way but works for me ;)

Thanks alot.

**Reply by Samuelopez-ansys** [reply]
@shizzlemaster The last merge into the PyAEDT main has added a new settings to modify the log file location :)

settings.aedt_log_file

**Reply by SMoraisAnsys** [reply]
@shizzlemaster See #5092 for more information !
